### PR TITLE
[FLOC-2314] Reduce env variable

### PR DESCRIPTION
**Pull request by adamtheturtle**
Fixes https://clusterhq.atlassian.net/browse/FLOC-2314

### CODE REVIEW

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 1]
```diff
@@ -115,9 +115,9 @@ Preparing For a Release
...
       # you want to continue connecting
       ssh-keyscan github.com >> ~/.ssh/known_hosts
-      git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
-      cd flocker-${VERSION}
-      mkvirtualenv flocker-release-${VERSION}
+      git clone git@github.com:ClusterHQ/flocker.git
+      cd flocker
+      mkvirtualenv flocker-release
       pip install --editable .[release]
       admin/create-release-branch --flocker-version="${VERSION}"
```
Comment (wallrj) [inline docs/gettinginvolved/infrastructure/release-process.rst]: :question: While doing the 1.0.0+doc1 release I wondered if would ever matter that you're install release tools from the master branch and expecting them to work on the release branch...

So this is the master version of `create-release-branch`...not much we can do about that, but....

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 2]
```diff
@@ -204,7 +205,7 @@ Preparing For a Release
 
    .. prompt:: bash [vagrant@localhost]$
 
-      ~/flocker-${VERSION}/admin/publish-docs --doc-version ${VERSION}
+      admin/publish-docs --doc-version ${VERSION}
```
Comment (wallrj) [inline docs/gettinginvolved/infrastructure/release-process.rst]: :question: ...this is now the version of `publish-docs` from the release branch....which may be very out-of-date if we're working a patch release of an old version.
Not an issue right now though.

**Comment by wallrj** [comment]
Thanks @adamtheturtle 

Having just performed a documentation-only release I can confirm that these changes would have worked and that these instructions would have been less confusing....for me.

Please merge.

File: docs/gettinginvolved/infrastructure/release-process.rst (hunk 1 above)
Comment (adamtheturtle) [inline docs/gettinginvolved/infrastructure/release-process.rst]: Agreed, this is a potential problem, but it is separate.

**Pull request merged**